packages/tokens/src/GovernedTokenWrapper.ts

```diff
@@ -5,6 +5,7 @@ import { GovernedTokenWrapper as GovernedTokenWrapperContract, GovernedTokenWrap
 export class GovernedTokenWrapper {
   contract: GovernedTokenWrapperContract;
   signer: ethers.Signer;
+  feeRecipient?: string;
```

Comment:
I suggest this to be `non-nullable` `readonly` and only passed to the constructor. This way it is hard to accidentally overwrite it during the runtime and cause confusion during testing.

Reply:
Instead of adding a `feeRecipient` instance variable. I just added a `getFeeRecipient` method to the GovernedTokenWrapper. That should do the trick in terms of getting the `feeRecipient` from a deployed bridge.

Reply:
Sounds good!